src/Microsoft.AspNet.TestHost/WebSocket.cs

```diff
@@ -0,0 +1,339 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.TestHost
+{
+    class WebSocket : System.Net.WebSockets.WebSocket
```

Comment:
@Tratcher Surely there's a way to reuse that code without copying the implementation here? We can shuffle the CI dependencies or move things to other repos if it causes problems.